pkg/imageMetadata/imageMetadata.go

```diff
@@ -0,0 +1,47 @@
+package imageMetadata
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/rwcarlsen/goexif/exif"
+)
+
+func GetImageCreationDate(imagePath string) (creationDate time.Time, Error error) {
+	file, err := os.Open(imagePath)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("error opening image: %v", err)
```

Comment:
Is it possible to return nil instead of time.Time{}, what is time.Time{}?

Reply:
Returning `nil` is not possible:
`cannot use nil as time.Time value in return statement`

My understanding was that in case we have an error, we should return an empty value (e.g an empty string `""`  or `0.0` for a float) together with the error.

For datetime, the best way I found to do this is to default construct a `time.Time` object, which has all its values (`day`, `month`, etc.) set to 0.